adafruit_io/adafruit_io.py

```diff
@@ -218,7 +218,7 @@ def remove_feed_callback(self, feed_key):
         validate_feed_key(feed_key)
         self._client.remove_topic_callback("{0}/f/{1}".format(self._user, feed_key))
 
-    def loop(self):
+    def loop(self, timeout=1):
```

Comment:
Please add a docstring :param: describing the timeout and the units (seconds? millis?) before the timeout expires